.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Description
+
+<!--
+Please include a summary of the changes and the related issue.
+Highlight any key points or areas of concern.
+-->
+
+<!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.*
+-->
+
+Fixes # (issue)
+
+### Is this user-facing behavior change?
+
+<!--
+If no, just leave this section empty.
+If yes, please explain how user-experience changes with this pull request.
+-->
+
+### Did you use Generative AI?
+
+<!--
+If this pull request was generated using Generative AI, specify the model (e.g., GitHub Copilot v3.2)
```

Comment:
Should we ask users to specify what part(s) of their PR's code was created using gen AI? Maybe just a hint, e.g., function X, class Y, or test Z. Or do we just ask yes/no?

Reply:
yeah let's do that. i think we can also add the example of model.